census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -315,6 +315,12 @@ public void streamCreated(Attributes transportAtts, Metadata headers) {
         headers.discardAll(tracingHeader);
         headers.put(tracingHeader, span.getContext());
       }
+      span.addAnnotation("Real stream created.");
```

Comment:
The existing annotation didn't include a period. We should do the same.

Is "real stream" meaningful to users? Should we instead say something like "Transport selected" or "Load balancer provided a transport"? Do we know what string C used here?

Reply:
C use  "Delayed name resolution complete."  and "Delayed LB pick complete."
It is a good idea to use the same descriptions, at least for the this new one.
https://github.com/grpc/grpc/pull/31913